docs/variants/protectli_vp46xx/hardware-matrix.md

```diff
@@ -54,7 +54,31 @@ coreboot port on the Protectli VP46XX firewall.
 | **Power Control**              | RTE v1.1.0                                  |
 | **Power supply**               | Channel Well Technology 12V, 7.5A 90W       |
 
-## Protectli VP4670
+## Protectli VP4670 ver1
+
+| Component                      | Description                                 |
+|--------------------------------|---------------------------------------------|
+| **CPU**                        | Intel Core i7-10810U                        |
+| **RAM**                        | Slot 1: Empty                               |
```

Comment:
@Psotas could we fill this slot? Platforms without enough RAM might work very very slowly.

Reply:
Ram swaped to:
| **RAM**                        | Slot 1: CRUCIAL CT16G4SFRA32A               |
|                                | Slot 2: CRUCIAL CT16G4SFRA32A               |